app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -911,7 +910,7 @@ public void onLocationChangedSignificantly(fr.free.nrw.commons.location.LatLng l
 
     @Override
     public void onLocationChangedSlightly(fr.free.nrw.commons.location.LatLng latLng) {
-        Timber.d("Location slightly changed");
+        Timber.d("Location significantly changed");
```

Comment:
this log is not in keeping with the name of this method, which one is happening here? Slightly or significantly?

Reply:
Slightly. I have fixed it. Thanks